modules/subscription/variables.tf

```diff
@@ -196,3 +196,81 @@ variable "wait_for_subscription_before_subscription_operations" {
 The duration to wait after vending a subscription before performing subscription operations.
 DESCRIPTION
 }
+
+variable "subscription_budgets" {
+  type = map(object({
+    amount            = number
+    time_grain        = optional(string, "Monthly")
+    time_period_start = string
+    time_period_end   = string
+    notifications = map(object({
+      enabled        = bool
+      operator       = string # EqualTo, GreaterThan, GreaterThanOrEqualTo
+      threshold      = number # 0-1000 percent
+      threshold_type = string # Actual, Forecasted
+      contact_emails = optional(list(string), [])
+      contact_roles  = optional(list(string), [])
+      contact_groups = optional(list(string), [])
+      locale         = optional(string, "en-us")
+    }))
+  }))
+  validation {
```

Comment:
Is it possible to extend the validation to include the notifications? Specifically `operator`, `threshold` and `threshold_type`?